install_config/aggregate_logging.adoc

```diff
@@ -817,6 +817,10 @@ additional parameters added:
 * `*ES_COPY_PASSWORD*`, `*OPS_COPY_PASSWORD*` - password to use to authenticate to
   Elasticsearch using username/password auth
 
+*Note:* Sending logs directly to an AWS Elasticsearch instance is not supported.  Please
+use xref:fluentd-external-log-aggregator[Fluentd Secure Forward] to understand how this
```

Comment:
I think we should simply state to use Fluentd Secure Forward in conjunction with a Fluentd instance that is configured with an AWS-ES plugin instead of "to understand how this may be accomplished"